goldens/foo/lib/declare_x.dart

```diff
@@ -8,3 +8,7 @@ import 'package:_test_macros/declare_x_macro.dart';
 class Foo {}
 
 void main() => print(Foo().x);
+
```

Comment:
should we be using `equalsIgnoringWhitespace`?

Reply:
Adding a blank line at the end is a bug in `_macro_tool`, already fixed in some other pending PR, I forget which :) reverted.